### ISSUE TITLE
(At least) two emote wheels

### ISSUE THREAD

**Issue by deferred-impact** [deferred-impact/battledudes-issue-tracker#45]
Season 8 dropped and... I ran out of emote slots.

Clearly more slots are needed. The most obvious reason is that there are two (or more) types of communication in a multiplayer game:
 - funny - friendly.png, thinking.png and such. Silly interactions that add charm to the game.
 - tactical - newly introduced emotes in S8 which allow discussing the strategy with your teammates.

The solution is obvious - let us set up 2 emote wheels. Assign each wheel to a different key. This not only lets us stuff more emotes (12, if you cannot count), but also lets us nicely group them as described above.

**Comment by Gregory-Y** [comment]
Good idea I guess. There's nothing that is particularly indicative of a bad idea so I'm willing to consider it.
